Assets/Mirror/Tests/Editor/NetworkWriterTest.cs

```diff
@@ -12,11 +12,13 @@ public void Benchmark()
         {
             // 10 million reads, Unity 2019.3, code coverage disabled
             //    4014ms ms
-            NetworkWriter writer = new NetworkWriter();
-            for (int i = 0; i < 10000000; ++i)
+            using (PooledNetworkWriter writer = NetworkWriterPool.GetWriter())
```

Comment:
please don't modify that benchmark. it's for testing raw networkwriter.
feel free to add a pooledwriter benchmark if you need one :)

Reply:
test restored